### PR TITLE
ticdc: update kafka minumum version

### PR DESCRIPTION
**Pull request by amyangfei**
### What is changed, added or deleted? (Required)

After test, the minimum Kafka version that CDC supports is `0.11.0.2`

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

**Comment by amyangfei** [comment]
PTAL @leoppro @TomShawn

File: ticdc/manage-ticdc.md [hunk 1]
```diff
@@ -166,7 +166,7 @@ URI 中可配置的的参数如下：
 | `127.0.0.1`          | 下游 Kafka 对外提供服务的 IP                                 |
 | `9092`               | 下游 Kafka 的连接端口                                          |
 | `cdc-test`           | 使用的 Kafka topic 名字                                      |
-| `kafka-version`      | 下游 Kafka 版本号（可选，默认值 `2.4.0`）                      |
+| `kafka-version`      | 下游 Kafka 版本号（可选，默认值 `2.4.0`，目前支持的最低版本为 `0.11.0.2`） |
```
Comment (leoppro) [inline ticdc/manage-ticdc.md:169]: 目前支持的最高版本为 2.6.0 （kafka 目前的最新版是2.6.0 我们是支持的，我测过）

**Comment by leoppro** [comment]
LGTM

File: ticdc/manage-ticdc.md (hunk 1 above)
Comment (amyangfei) [inline ticdc/manage-ticdc.md:169]:
```suggestion
| `kafka-version`      | 下游 Kafka 版本号（可选，默认值 `2.4.0`，目前支持的最低版本为 `0.11.0.2`，最高版本为 `2.6.0`） |
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**